assets/src/animation/outputs/animationOutput.js

```diff
@@ -33,9 +33,38 @@ function AnimationOutput({ config }) {
         type="application/json"
         dangerouslySetInnerHTML={{
           __html: JSON.stringify(
-            configs.map((conf) => ({
-              ...conf,
-            }))
+            // Only print the actual items needed for the animations,
+            // as defined by the prop types.
+            // Prevents printing the `element` prop which `config` can have.
+            configs.map(
+              ({
+                selector,
+                animation,
+                keyframes,
+                delay,
+                direction,
+                duration,
+                easing,
+                easingPreset,
+                endDelay,
+                fill,
+                iterationStart,
+                iterations,
+              }) => ({
+                selector,
+                animation,
+                keyframes,
+                delay,
+                direction,
+                duration,
+                easing,
+                easingPreset,
+                endDelay,
+                fill,
+                iterationStart,
+                iterations,
+              })
+            )
```

Comment:
Ideally we'd avoid this, ensuring `element` is never passed down here.

@littlemilkstudio Perhaps you can help here?

Reply:
Maybe I'll shorten the list a bit, but it would definitely be good to fix the root cause to  not pass down `element` in the first place.

Reply:
yea completely agree. I don't think we're using `endDelay`, `animation` or `iterationStart` but those are all valid properties, so fine to leave in there. Ope, just noticed `easingPreset` is in there. That's just an internal property that maps to predefined bezier curves. Fine to remove that one